src/components/new-request/name-check/resources/name-check-base-items-conflicts.ts

```diff
@@ -0,0 +1,145 @@
+import { NameCheckErrorType, NameCheckItemType } from '@/enums'
+
+export const baseItemsConflicts = {
+  'errorExact': {
```

Comment:
I'd love to see an enum used for these keys.

Reply:
sure thing

Reply:
Looks good!